charts/traction/values.yaml

```diff
@@ -193,40 +189,46 @@ acapy:
 
 postgresql:
   # when overriding global.fullnameOverride, you must override this to match.
-  fullnameOverride: "traction"
+  fullnameOverride: ""
 
-  image:
-    tag: 12
+  auth:
+    existingSecret: '{{ include "global.fullname" . }}'
+    secretKeys:
+      adminPasswordKey: postgres-password
+      userPasswordKey: database-password
+    # -- PostgreSQL Database to create.
+    database: traction
+    username: acapy
```

Comment:
@WadeBarnes any opinions on whether to use `acapy` or `traction` as username for the database? It appears that generating a random username might not be too straightforward